turbinia/config/__init__.py

```diff
@@ -213,60 +211,55 @@ def __init__(self, recipe_file):
     self.name = ""
     self.jobs_allowlist = []
     self.jobs_denylist = []
-    self.filter_patterns = []
     self.task_recipes = {}
 
   def load(self):
     """ Load recipe from file. """
     LoadConfig()
-    with open(self.recipe_file, 'r') as r_file:
-      try:
-        recipe_file_contents = r_file.read()
-      except yaml.parser.ParserError as exception:
-        message = (
-            'Could not load recipe file {0:s}: {1!s}'.format(
-                self.recipe_file, exception))
-        log.error(message)
-        raise TurbiniaException(message)
-        sys.exit(1)
-
-    recipe_dict = load(recipe_file_contents, Loader=Loader)
+    try:
```

Comment:
Maybe we can flatten this out and just have one `try` block with two `except` clauses for the two different exception types here?

Reply:
Agreed.